static/js/board.js

```diff
@@ -158,59 +197,164 @@ function createBoardElement(bob) {
+    }
+    else{
+      eventTime = null;
+    }
+
+    switch(eventTime){
+      case 'Today':
+        $eventsToday.append($newEvent);
+        broadcastText += eventsData[i].title;
+        broadcastText += "@" + Date.parse(eventsData[i].start).toString("HH:mm");
+        break;
+      case 'Tomorrow':
+        $eventsTomorrow.append($newEvent);
+        break;
+      case 'ThisWeek':
+        $eventsThisWeek.append($newEvent);
+        break;
+    }
+  }
+  console.log(broadcastText)
+  $broadcast.append($("<p/>", {text: broadcastText}) );
+}
 
+/**
+  * Creates html element for an event parsed from ABE.com
+  * @param {Object} eventData - A single JSON instance of ABE event
+*/
+function createEventObject(eventData) {
+  var $html = $('<li>', {
+    id: eventData.id,
+    class: "collection-item"
+  })
+    .append($('<span>', { class: 'title', text: eventData.title }))
+    .append($('<p>', { class: 'date', text:  Date.parse(eventData.start).toString("HH:mm ddd, MMMM dd") +" @ " + eventData.location}))
+    .append($('<p>', { class: 'description', text: eventData.description }));
   return $html;
 }
 
 /**
- * Defines time interval for carousel auto slides
- */
-$(function() {
-  setInterval(function() {
-    $('#slideshow').carousel('next');
-  }, 10000);
-  setInterval(function() {
-    $('#slideshow-small').carousel('next');
-  }, 7000);
-});
+  * Updates the caraousel item with updated bob data
+  * Called when update_element socket received
+  * @param {Object} bobData - contains updated bob data
+*/
+function updateBoardElement(bobData){
+  var bobId = bobData._id;
+  var $bobToUpdate = $("#" + bobId);
+  var $imageHolder = $bobToUpdate.find(".image-holder");
+  var newImage = "background-image: url(" + bobData.data.Link + ")";
+  $imageHolder.attr("style", newImage);
+  var $textHolder = $bobToUpdate.find(".text-holder");
+  $textHolder.find(".author").attr("text", bobData.Title);
+  $textHolder.find(".description").attr("text", bobData.Description);
+}
 
 /**
- * Changes active item to either previous or next item depending on direction
- * @param {string} direction - direction of moving : left, right
+  * Deletes the html bob element with bobid from carousel
+  * @param {string} bobid - id of the bob to be deleted
+*/
+function deleteElement(bobid){
+  var $bobToDelete = $("#" + bobid);
+  $bobToDelete.remove();
+}
+
+/**
+  * Updates the label with the votes from bob with bobid
+  * @param {string} bobid - id of the bob of the votes
+*/
+function updateVoteLabel(bobid){
+  var $labelToUpdate = $("#votes");
+  var votes =  $.get('/api/bobs/' + bobid + "/votes", function(res){
+    $labelToUpdate.attr("data-badge-caption", "+" + res.votes);
+  });
+}
+
+/**
+  * Updates the label with the votes returned from socket
+  * parpm {object} res - response from the socekt which contains bobid and votes
+*/
+function incrementVote(res){
+  var $labelToUpdate = $("#votes");
+  $labelToUpdate.attr("data-badge-caption", "+" + res.votes);
```

Comment:
I think this is cleaner in one line.
$("#votes").attr("data-badge-caption", "+" + res.votes);